website/www/site/layouts/partials/head.html

```diff
@@ -41,10 +41,7 @@
 <script src="/js/section-nav.js"></script>
 <script src="/js/page-nav.js"></script>
 
-{{/* 
-  <link rel="alternate" type="application/rss+xml" title="{{ .Site.Title }}" href="/feed.xml">
-*/}}
-  
+<link rel="alternate" type="application/rss+xml" title="{{ .Site.Title }}" href="/index.xml">
```

Comment:
We need to do some research on handling RSS in hugo. As you can see in feed.xml from Jekyll, they also put the latest 10 blogposts in the feed. We need to do the same (in the future of course)